usrsctplib/meson.build

```diff
@@ -1,9 +1,9 @@
 # Includes
-include_dirs = include_directories(
+include_dirs = [include_directories(
```

Comment:
I'm wondering if this isn't already an array. Have you tested it without the brackets?

Reply:
Yes, I tested it without the brackets and that didn't work (even though I expected it to). Unlike `files()`, `include_directories()` doesn't implement the `+=` operator. I'm not sure why.

Reply:
I looked at the parser, and the reason is that `files()` returns an array of `File` objects, while `include_directories()` contains multiple include dirs in a single object. Could be fixed upstream.